tests/page-objects/EditorPage.ts

```diff
@@ -43,24 +43,34 @@ export class EditorPage {
         await this.addOpposingReviewer();
     }
 
-    private async addPersonToPeoplePicker(picker: Selector): Promise<void> {
+    private async addPersonToPeoplePicker(picker: Selector, number = 1): Promise<void> {
         await t.expect(picker.find('.selected_people_list__item').count).eql(1);
         const addButton = picker.find('.pod__button');
         await t.click(addButton);
+        await this.assertPeoplePickerSearch();
```

Comment:
merge issue, this should be removed again.

Reply:
done